### ISSUE TITLE
SQL parameters are not logged

### ISSUE THREAD

**Issue by axelson** [timberio/timber-elixir#289]
With `{Timber.Integrations.EctoLogger, :log, []}` listed in the `:loggers` key for my Ecto Repository I see log statements like this:

```
[debug] Processed SELECT c0."id", c0."name" FROM "communities" AS c0 WHERE (c0."id" = $1) in 3ms
```

But I would expect to see the value of the parameters (in this case it is `$1`). Is there a reason that they are hidden? Can they be added back? (note: by default they are logged by Ecto).

**Comment by binarylogic** [comment]
Hey @axelson, great request / issue. `Ecto` constructs what are called [prepared statements](https://www.postgresql.org/docs/9.3/static/sql-prepare.html) which are primarily used to protect against SQL injections. This is why you see `$1`, `$2`, etc. What `Ecto` does provide are the [query parameters](https://hexdocs.pm/ecto/Ecto.LogEntry.html). We can make that option to include as metadata for the `sql_query` event. This way you can see the values when you inspect the log line. Would that be sufficient?

**Comment by axelson** [comment]
Yes that would be great since that is similar to what Ecto logs by default.
